scripts/globals/sparkshop.lua

```diff
@@ -83,7 +83,7 @@ local optionToItem = {
         [19] = { cost = 50, id = 16966 }, -- Tachi
         [20] = { cost = 50, id = 17024 }, -- Ash club
         [21] = { cost = 50, id = 17034 }, -- Bronze mace
-        [22] = { cost = 50, id = 4181  }, -- Bronze hammer
+        [22] = { cost = 50, id = 17042 }, -- Bronze hammer
         [23] = { cost = 50, id = 17042 }, -- Bronze rod
```

Comment:
Checked, and 17042 is correct, but can you please fix Bronze Rod then as well? 17059, but its the same value across two lines.

Reply:
good catch